content/tutorials/portenta-h7/por-ard-flash/content.md

```diff
@@ -1,6 +1,6 @@
 # Using the Flash on Portenta H7 – MCU
 
-This tutorial shows you how to use the internal flash memory of the microcontroller of Portenta H7 and other MbedOS-based Arduino boards (Nano 33 BLEs, etc.). This is Part 1 of a series of projects on how to use the flash memories on the MbedOS-based Arduino boards.
+In this tutorial we will use the internal flash memory of the microcontroller of Portenta H7 and other MbedOS-based Arduino boards (Nano 33 BLE andNano 33 BLE Sense). This is Part 1 of a series of projects on how to use the flash memories on the MbedOS-based Arduino boards. 
```

Comment:
I am not 100% sure about adding the last sentence here, I would remove the "This is Part 1 of ..." since it may happen that it takes a while till we have the next parts